bpdv3/Assets/Scripts/Lose Condition/BottomRowHazard.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Scripts.Player;
+﻿using Scripts.A;
```

Comment:
I thought naming it Scripts.Player would be different to naming it Player. But I'm getting the impression this is not the case. Thanks for bringing to my attention. I shall rename it with a more suitable name in the near future. Thanks for bringing to my attention.